Formula/corectl.rb

```diff
@@ -13,27 +22,49 @@ class Corectl < Formula
 
   depends_on "go" => :build
   depends_on "godep" => :build
+  depends_on "ocaml" => :build
+  depends_on "opam" => :build
   depends_on :macos => :yosemite
 
   def install
     ENV["GOPATH"] = buildpath
 
+    opamroot = buildpath/"opamroot"
+    opamroot.mkpath
+    ENV["OPAMROOT"] = opamroot
+    ENV["OPAMYES"] = "1"
+
     path = buildpath/"src/github.com/TheNewNormal/#{name}"
     path.install Dir["*"]
 
     args = []
     args << "VERSION=#{version}" if build.stable?
 
     cd path do
-      system "make", "corectl", *args
-      system "make", "documentation/man"
-      bin.install "corectl"
+      system "opam", "init", "--no-setup"
+      system "opam", "pin", "add", "qcow-format",
+        "https://github.com/mirage/ocaml-qcow.git#master"
```

Comment:
Any chance we can pin this to a specific commit rather than `master`? Note: I only use `opam` minimally, so if this comment makes no sense, feel free to say 😉.